src/frontend/App.tsx

```diff
@@ -78,14 +77,12 @@ const App = () => {
   }, []);
 
   return (
-    <IntlProvider>
```

Comment:
this is an extra intl provider that isn't needed (already set up in AppProviders)